packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1025,6 +1042,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         onMouseEnter={ this._onOptionMouseEnter.bind(this, item.index) }
         onMouseMove={ this._onOptionMouseMove.bind(this, item.index) }
         onMouseLeave={ this._onOptionMouseLeave }
+        onKeyDown={ this._onListItemKeyDown(item.index) }
```

Comment:
I think this bit of code reintroduces the issue where you have to tab through each item in the combobox to get to the next element.  Not sure that we should ever have tab focus on the combobox elements

Reply:
My PR is trying to close the gap between fabric repo js and control repo js. This is the bit they added. Jiahong might have better idea